README.md

```diff
@@ -15,27 +15,23 @@ import (
 )
 
 func main() {
-    f1.Execute()
+    f1.Scenarios().Execute()
```

Comment:
Since this doesn't do anything useful, I would skip this whole section "Building an executable binary". The next section also covers this well enough.